src/shims/macos/nmp.ts

```diff
@@ -0,0 +1,63 @@
+import { platform } from 'node:os';
```

Comment:
- [ ] Please verify whether this is a valid way to check for the platform. I saw a web build, and `state.ts` uses `importNodeMacPermissionsOrShim`, so I'm not sure if the `node:` prefix is okay and if the import is even available.